src/rpc/net.cpp

```diff
@@ -303,15 +303,17 @@ static RPCHelpMan addconnection()
         "\nOpen an outbound connection to a specified node (test only)",
         {
             {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The IP address and port to attempt connecting to."},
+            {"connectiontype", RPCArg::Type::STR, RPCArg::Optional::NO, "'outbound' or 'blockrelay'"},
```

Comment:
0fb4635bbabb1bea10a215bde4c803ceb5bc5f26
Why not keep consistency about full stops at the end of argument descriptions?

Reply:
fixed